lumigator/backend/backend/services/jobs.py

```diff
@@ -163,7 +163,6 @@ def generate_config(self, request: JobCreate, record_id: UUID, dataset_path: str
                 base_url=request.job_config.base_url if request.job_config.base_url else None,
                 model=request.job_config.model,
                 provider=request.job_config.provider,
-                system_prompt=request.job_config.task_definition.system_prompt,
```

Comment:
The system prompt is passed in to the `InferenceJobConfig` up in line 138, so the system_prompt is still accessible tot he job.